crates/fluvio/src/lib.rs

```diff
@@ -240,6 +240,10 @@ pub mod metadata {
         pub use fluvio_sc_schema::objects::*;
     }
 
+    pub mod tableformat {
+        pub use fluvio_sc_schema::tableformat::*;
```

Comment:
need to bump up fluvio version.  also add to changelog

Reply:
The version as already bumped in a previous PR, I will add to the changelog

Reply:
do you mean minor bump?

Reply:
Yes